apps/backend-ops/src/firestoreMigrations/0023.ts

```
@@ -0,0 +1,89 @@
+import { Firestore } from '../admin';
+
+
+export async function upgrade(db: Firestore) {
+  const invitations = await db.collection('invitations').get();
+  const notifications = await db.collection('notifications').get();
+  const orgs = await db.collection('orgs').get();
+  const batch = db.batch();
+
+  // Update invitations with organization
+  invitations.docs.forEach(doc => {
+    const invitation = doc.data();
+    const { organizaton } = invitation;
+
+    if (organizaton) {
+      delete invitation.organizaton;
+      const newData = {
+        ...invitation,
+        organization: organizaton,
+      };
+
+      if (!newData) {
+        console.error(doc.ref.path, invitation, newData);
+      }
+
+      return batch.set(doc.ref, newData);
+    }
+  });
+
+  // Update notification with organization
+  notifications.docs.forEach(doc => {
+    const notification = doc.data();
+    const { organizaton } = notification;
+
+    if (organizaton) {
+      delete notification.organizaton;
+      const newData = {
+        ...notification,
+        organization: organizaton,
+      };
+
+      if (!newData) {
```

Is this check important? If so maybe you should throw an error because as it is right now, you will try to set an empty doc in `return batch.set(doc.ref, newData);`